selectrum.el

```diff
@@ -887,7 +823,10 @@ into the user input area to start with."
   (add-hook
    'minibuffer-exit-hook #'selectrum--minibuffer-exit-hook nil 'local)
   (setq-local selectrum--init-p t)
-  (setq selectrum--minibuffer (current-buffer))
+  (setq selectrum--candidates-overlay
+        (make-overlay (point) (point) nil t t))
```

Comment:
```suggestion
        (make-overlay (point) (point) nil 'front-advance 'rear-advance))
```

Just a suggestion, it's fine if you don't like this style.

Reply:
Thanks, I like that style better, too.